cppcache/integration-test/CacheHelper.cpp

```diff
@@ -1848,16 +1851,17 @@ int CacheHelper::getRandomAvailablePort() {
 }
 
 std::string CacheHelper::unitTestOutputFile() {
-  char currWDPath[512];
```

Comment:
Looks brittle.  If only we had C++17, then we'd also have `std::filesystem`... [sigh]

Reply:
Someday @dgkimura someday

Reply:
We could use boost::filesystem here. I just wanted to make it less nasty than it was to move forward since this is framework code.